src/v2/Apps/Artist/Routes/AuctionResults/Components/AuctionFilters/index.tsx

```diff
@@ -1,14 +1,15 @@
 import React from "react"
 import { AuctionHouseFilter } from "./AuctionHouseFilter"
 import { MediumFilter } from "./MediumFilter"
-import { SizeFilter } from "./SizeFilter"
+import { SizeFilter } from "v2/Apps/Components/SizeFilter"
 import { YearCreated } from "./YearCreated"
+import { useAuctionResultsFilterContext } from "../../AuctionResultsFilterContext"
 
 export const AuctionFilters: React.FC = () => {
   return (
     <>
       <MediumFilter />
-      <SizeFilter />
+      <SizeFilter useFilterContext={useAuctionResultsFilterContext} />
```

Comment:
The nature of hooks is such to where you don't need to pass them as props; from inside Size filter can do `

```jsx
const filterContext = useFilterContext()
```

And things will be pulled from the context.

Reply:
But the problem is that this SizeFilter component is shared, and we don't have a shared context. So in this case, the way I use the hook passed as a prop is a tiny bit unorthodox, but I think it makes sense for here. In the TODO I added, if we have a "base" filter we can also have a "base" filter context that we can use the hook from. For now, I think the only other way to do this is to give some other unique value as a prop and then in the SizeFilter it can pull from the auction filter context or from the artwork filter context.

I kinda prefer the hook-as-prop uniqueness. What do you think?

Reply:
We should avoid props passing if possible because it reduces the bindings between things, but i see what you mean about the two separate contexts; might be a good example of over engineering to make that work when a simple prop suffices 👍